service/src/main/java/com/fitgoal/service/AuditService.java

```diff
@@ -0,0 +1,19 @@
+package com.fitgoal.service;
+
+import com.fitgoal.dao.domain.AuditDto;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface AuditService {
```

Comment:
We had a conversation that this service:
1. Has much more methods than it makes sense for it to have.
2. Can be eliminated since it does nothing compare to `AuditDAO` and it's fine to inject `AuditDAO` into `NotificationServiceImpl`.

Are you going to address this?

Reply:
1. This issue is already fixed. I pushed changes to **develop** branch (https://github.com/YulianaYershova/notification_service/blob/develop/service/src/main/java/com/fitgoal/service/AuditService.java) but forgot to merge develop **branch** into **mongodb**. 

2. Refactored.